components/clp-package-utils/clp_package_utils/scripts/start_clp.py

```diff
@@ -491,6 +491,7 @@ def start_results_cache(instance_id: str, clp_config: CLPConfig, conf_dir: pathl
     cmd = [
         "docker", "run",
         "-d",
+        "--network", "host",
```

Comment:
Just curious, do you know exactly why we need --network host? is there any IP other than results_cache.port that will be accessed?